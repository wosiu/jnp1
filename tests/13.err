Error 12: 13 1.1.2000 12.00 10
Error 13: 14 1.1.2000 13.00 10
Error 14: 15 1.1.2000 14.00 10
Error 15: 16 1.1.2000 15.00 10
Error 16: 17 1.1.2000 15.00 10
Error 17: 18 1.1.2000 15.00 10
Error 18: 19 1.1.2000 16.00 10
Error 26: 11 1.1.2000 10.00
Error 27: 12 1.1.2000 11.00
Error 35: 7 1.1.2000 6.00
