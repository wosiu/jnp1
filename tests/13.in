1 1.1.2000 0.00
2 1.1.2000 1.00 20
3 1.1.2000 2.00
4 1.1.2000 3.00 30
5 1.1.2000 4.00
6 1.1.2000 5.00 40
9 1.1.2000 8.00
10 1.1.2000 9.00 20
11 1.1.2000 10.00
12 1.1.2000 11.00
L 0.00 0.00
13 1.1.2000 12.00 10
14 1.1.2000 13.00 10
15 1.1.2000 14.00 10
16 1.1.2000 15.00 10
17 1.1.2000 15.00 10
18 1.1.2000 15.00 10
19 1.1.2000 16.00 10
L 0.00 0.30
L 1.20 1.20
L 0.30 1.30
L 0.00 2.00
L 0.00 2.30
L 0.00 3.00
L 0.00 3.30
11 1.1.2000 10.00
12 1.1.2000 11.00
L 0.00 4.00
L 4.00 9.00
L 4.30 9.00
L 5.00 9.00
L 5.30 9.00
L 6.00 9.00
M 0.00 2.00
7 1.1.2000 6.00
