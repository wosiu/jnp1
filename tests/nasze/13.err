Error 12: 013 1.1.2000 12.00 10
Error 13: 014 1.1.2000 13.00 10
Error 14: 015 1.1.2000 14.00 10
Error 15: 016 1.1.2000 15.00 10
Error 16: 017 1.1.2000 15.00 10
Error 17: 018 1.1.2000 15.00 10
Error 18: 019 1.1.2000 16.00 10
Error 26: 011 1.1.2000 10.00
Error 27: 012 1.1.2000 11.00
Error 35: 007 1.1.2000 6.00
