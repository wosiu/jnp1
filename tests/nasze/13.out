1
1
1
1
3
3
3
4
5
3
2
2
2
1
20
